plugin/documents.py

```diff
@@ -267,9 +280,8 @@ def on_selection_modified_async(self) -> None:
             self.update_diagnostic_in_status_bar_async()
 
     def on_post_save_async(self) -> None:
-        if self.view.is_primary():
-            for sv in self.session_views_async():
-                sv.on_post_save_async()
+        for sv in self.session_views_async():
+            sv.on_post_save_async()
```

Comment:
The notifications triggered in SessionBuffer (like DidSave) will trigger twice for cloned views now.

Reply:
I decided that it was better to "force-register" clones, if there exist any, instead of this old approach from 2 days ago.